src/components/Login.vue

```diff
@@ -52,8 +49,7 @@ export default {
         email: "",
         password: ""
       }
-    };
-  },
+    },
```

Comment:
Need to add one more brace here before this `},`
The `data()` function is separate form the `methods:` key.

Reply:
Ok resolved. Thanks for the catch.